dashboard/test/controllers/home_controller_test.rb

```diff
@@ -297,9 +297,9 @@ class HomeControllerTest < ActionController::TestCase
     get :home
 
     assert_select '#student-information-modal'
-    assert_select '#user_age'
-    assert_select '#user_us_state', false
-    assert_select '#user_gender_student_input', false
+    assert_select '#student_age'
```

Comment:
Rails prepends the user class for these attributes because the form builder knows the object is a User model. However since a user is now either a Teacher or Student class, these attributes are now being prepended with `student_` rather than `user_`.